WordPress/Classes/ViewRelated/Site Creation/Design Selection/SiteDesignContentCollectionViewController.swift

```diff
@@ -52,26 +35,29 @@ class SiteDesignContentCollectionViewController: FilterableCategoriesViewControl
                 SiteDesignSection(category: category, designs: siteDesigns.designs.filter { design in design.categories.map({$0.slug}).contains(category.slug)
                 })
             }
-            NSLog("sections: %@", String(describing: sections))
             contentSizeWillChange()
             tableView.reloadData()
         }
     }
 
-    var selectedDesign: RemoteSiteDesign? {
-        guard let sectionIndex = selectedItem?.section, let position = selectedItem?.item else { return nil }
-        return sections[sectionIndex].designs[position]
+    var selectedPreviewDevice: PreviewDevice {
+        get { .mobile }
+        set { /* no op */ }
     }
 
     init(createsSite: Bool, _ completion: @escaping SiteDesignStep.SiteDesignSelection) {
         self.completion = completion
         self.createsSite = createsSite
+        tableView = UITableView(frame: .zero, style: .plain)
+        tableView.separatorStyle = .singleLine
+        tableView.separatorInset = .zero
+        tableView.showsVerticalScrollIndicator = false
 
         super.init(
-            analyticsLocation: "site_creation",
+            scrollableView: tableView,
             mainTitle: TextContent.mainTitle,
-            primaryActionTitle: createsSite ? TextContent.createSiteButton : TextContent.chooseButton,
-            secondaryActionTitle: TextContent.previewButton
+            // the primary action button is never shown
+            primaryActionTitle: ""
```

Comment:
This is the only part where `CollapsableHeaderViewController` didn't quite support what we needed. We'll never show a primary action, so there's no need for us to supply a title.